src/azul/indexer/document.py

```diff
@@ -302,6 +311,7 @@ class NullableNumber(Generic[N_], FieldType[Optional[N_], Number]):
     null_int = sys.maxsize - 1023
     assert null_int == int(float(null_int))
     es_type = 'long'
+    filter_operators = ('is', 'within')
```

Comment:
This enables filtered for numerical values between a range.
i.e. `cellCount between 5 and 20`